src/lib/MediaQueryContext.tsx

```diff
@@ -11,7 +11,7 @@ const MediaQueryContext = React.createContext({
 
 export interface MediaQueryProviderProps {
   children: React.ReactElement;
-  mediaQueryBreakPoint?: string | boolean;
+  mediaQueryBreakPoint?: string;
```

Comment:
string | boolean is intended~~ Actually only having boolean is better ~

Reply:
ok, I reverted the changes to this file